modules/admin/proc-customizing-telemetry-segment-using-helm.adoc

```diff
@@ -0,0 +1,37 @@
+[id="proc-customizing-telemetry-segment-using-helm_{context}"]
+= Customizing telemetry Segment source using the Helm Chart
+
+You can configure integration with your Segment source by using the Helm Chart.
```

Comment:
Integration with RHDH?

Reply:
Here, we are not configuring integration with our product, it is more of integration with that telemetry entity, that is, Segment source. By default, the collected telemetry data is sent to Red Hat. But, if a user wants to change this  setting and send telemetry data to some other location. Then, this integration is needed.